NET-1825: More new ACL token creation docs (#18063)
hashicorp/consul: website/content/docs/security/acl/tokens/create/create-a-replication-token.mdx

---
layout: docs
page_title: Create tokens for service registration
description: >-
  Learn how to create ACL tokens that a server agent in a secondary datacenter can use for ACL token replication between WAN-federated datacenters.
---

# Create a replication token

> This topic describes how to configure an ACL token for ACL replication between WAN-federated datacenters.

Reviewer: ```suggestion
This topic describes how to configure an ACL token for ACL replication between WAN-federated datacenters. If your Consul clusters are connected through peer connections, ACL replication is not required. To learn more about cluster peering, refer to the [comparison between WAN federation and cluster peering](/consul/docs/connect/cluster-peering#compared-with-wan-federation).
```

## Introduction

> Consul agents must present a token linked to policies that grant the appropriate set of permissions.
> Specify the [`replication`](/consul/docs/agent/config/config-files#acl_tokens_replication) token on each server in a non-primary datacenter. To learn about configuring ACL replication, refer to the ACL Replication for Multiple Datacenters tutorial.
> 
> -> Note: ACL replication is only supported for WAN-federated datacenters. When using cluster peering to connect Consul datacenters, ACL replication is not required. To learn more about cluster peering, refer to Differences between WAN federation and cluster peering.

Reviewer: ```suggestion
Consul agents must present a token linked to policies that grant the appropriate set of permissions.
Specify the [`replication`](/consul/docs/agent/config/config-files#acl_tokens_replication) token on each server in a non-primary datacenter. For hands-on instructions on how to configure ACL replication across datacenters, refer to the [ACL Replication for Multiple Datacenters](/consul/tutorials/security-operations/access-control-replication-multiple-datacenters) tutorial.

```
Moved the statement about WANFED vs CP to the description so that people can bounce immediately if this topic isn't for them.
